cmd/aspect/main.go

```diff
@@ -45,6 +45,7 @@ func main() {
 	defer pluginSystem.TearDown()
 
 	cmd := root.NewDefaultRootCmd(pluginSystem)
+	cmd = pluginSystem.AddCustomCommands(cmd)
```

Comment:
Why do this here instead of in `NewDefaultRootCmd` where (I assume?) the rest of the plugin-data-extraction is done?

Reply:
I think you're right. Thats probably a better spot for it